pkg/updater/repository/link.go

```diff
@@ -0,0 +1,37 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+package repository
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/google/renameio/v2"
+)
+
+func linkRead(linkPath string) (string, error) {
+	return filepath.EvalSymlinks(linkPath)
+}
+
+func linkExists(linkPath string) (bool, error) {
+	_, err := os.Stat(linkPath)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
I believe the `err != nil` is useless here.
You might want to use `errors.Is(err, fs.ErrNotExist)` instead of `os.IsNotExist`, even the doc of `os.IsNotExist` advises so

Reply:
sounds good 👍🏻

Reply:
Done ✔️